<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GwmLayoutBatchDialog</class>
 <widget class="QWidget" name="GwmLayoutBatchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GwmLayoutBatchDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,2,0" columnstretch="0,0,0,0,1">
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Select Fields</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QListView" name="lsvField"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4" rowspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Configure Symbol</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="btnEditSymbol">
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/images/themes/default/symbologyEdit.svg</normalon>
           </iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnUnifyRow">
          <property name="text">
           <string>Unify Row</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/images/themes/default/mActionAlignVCenter.svg</normalon>
           </iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnUnifyCol">
          <property name="text">
           <string>Unify Column</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/images/themes/default/mActionAlignHCenter.svg</normalon>
           </iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTreeView" name="trvSymbology"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Select Layers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QListView" name="lsvLayer"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Select Layout</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QListView" name="lsvLayout"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
